src/components/Experts/Experts.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './style.scss';
+import StarIcon from '@material-ui/icons/Star';
+import StarOutlineIcon from '@material-ui/icons/StarOutline';
+
+function Experts() {
+  return (
+    <section className="experts">
+      <div className="experts__container">
+        <div className="experts__header">
+          <h5>Practice Advice</h5>
+          <h1>Our Experts Teacher</h1>
+          <p>
+            Problems trying to resolve the conflict between
+            <br />
+            the two major realms of Classical physics: Newtonian mechanics
+          </p>
+        </div>
+        <div className="experts-review__box">
+          <div className="expert-review__block">
+            <div className="expert__picture expert__picture--first" />
```

Comment:
picture related to --first and --second are definitely content - as they show real data about real people description below is referring to